cattle/compute/__init__.py

```diff
@@ -63,6 +47,27 @@ def instance_deactivate(self, req=None, instanceHostMap=None,
                                                         progress)
         )
 
+    def get_instance_host_from_map(self, instanceHostMap=None):
+        instance = instanceHostMap.instance
+        host = instanceHostMap.host
+
+        try:
+            host.clusterConnection = \
+                instanceHostMap.data.fields['clusterConnection']
+            log.debug('clusterConnection = %s', host.clusterConnection)
+            host.caCrt = \
+                instanceHostMap.data.fields['caCrt']
+            host.clientCrt = \
+                instanceHostMap.data.fields['clientCrt']
+            host.clientKey = \
+                instanceHostMap.data.fields['clientKey']
```

Comment:
@sonchang Son, do you think we should disable logging for the cert information? As its a sensitive info